var/spack/repos/builtin/packages/crtm-fix/package.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class CrtmFix(Package):
+    """CRTM coeffecient files"""
+
+    homepage = "https://github.com/NOAA-EMC/crtm"
+    url = "ftp://ftp.ucar.edu/pub/cpaess/bjohns/fix_REL-2.3.0_emc.tgz"
+
+    maintainers = [
+        "BenjaminTJohnson",
+        "edwardhartnett",
+        "AlexanderRichert-NOAA",
+        "Hang-Lei-NOAA",
+        "climbfuji",
+    ]
+
+    version("2.4.0", sha256="7924285b8aa25b0b864643f03e7f281ca1816958e24c76aa9531b3ca902bf6e9")
+    version("2.4.0_emc", sha256="88d659ae5bc4434f7fafa232ff65b4c48442d2d1a25f8fc96078094fa572ac1a")
+    version("2.3.0_emc", sha256="fde73bb41c3c00666ab0eb8c40895e02d36fa8d7b0896c276375214a1ddaab8f")
```

Comment:
Similar concern to crtm package versions: If I add @2.4.0 and @2.4.0_emc to my environment, @2.4.0 concretizes to @2.4.0_emc.

Reply:
Those `crtm-fix` packages were added by @kgerheiser for NOAA-EMC, we don't use them at all. I'd be happy to remove this additional "package" entirely, unless it's required by some folks at NOAA.